setup.cfg

```diff
@@ -11,13 +11,8 @@ addopts =
     --tb native
     -W ignore
     --cov=alibi_detect
-    --randomly-dont-reorganize
```

Comment:
Why do we remove this here?

Reply:
For `--randomly-seed=0`, I've moved it to `ci.yml` because having it in the global pytest config caused an error in the notebook CI because `pytest-randomly` is now disabled there.

For `--randomly-dont-reorganize`, I decided we could remove it anyway. This controls whether or not the order of tests is randomly reorganized. From the `pytest-randomly` docs:

> By randomly ordering the tests, the risk of surprising inter-test dependencies is reduced - a technique used in many places, for example Google’s C++ test runner [googletest](https://code.google.com/p/googletest/wiki/V1_5_AdvancedGuide#Shuffling_the_Tests). Research suggests that “dependent tests do exist in practice” and a random order of test executions can effectively detect such dependencies [[1]](https://pypi.org/project/pytest-randomly/#footnote-1).

Reply:
btw we can add `--randomly-dont-reorganize` to `ci.yml` if you think this change shouldn't be made without further discussion...

Reply:
Ah yeah, I was curious about `--randomly-dont-reorganise`, it seems useful no? I suppose because we don't really deal with persistent state it's not the type of thing that's going to help us much. Although perhaps it's worth keeping in for the saving tests, that's the only place where I can image one test leaking into another...

Reply:
Yeh agree that I can't think of any obvious places where tests might be leaky. I hope that the saving ones are ok since they use pytest's [tmp_path](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html). I sort of think if we keep `pytest_randomly` it doesn't hurt to randomise test order, but if we're removing it its not worth implementing an alternative solution to randomise tests.

Reply:
yeah, aggreed

Reply:
Removing `--randomly-dont-reorganise` actually enables random order, right? In which case it seems useful to have?

Reply:
Yep, random order has been enabled by removing the flag.

Reply:
@jklaise just to confirm, are you saying random order seems useful to have? i.e. we stick to having the flag removed...

Reply:
Correct

Reply:
OK thanks. Think this is good to merge then 👍🏻